README.md

```diff
@@ -198,9 +198,16 @@ optional `prepare.yml` and `verify.yml` playbooks. Running the tests then
 proceeds as with testing the roles:
 
 ```shell
-molecule --base-config ../molecule_configs/centos7_base_config.yml test --scenario centos7_monitoring
+molecule test --scenario centos7_monitoring
 ```
 
+Alternative you can use `molecule converge` to avoid destroying the container,
+as [described above](#inspecting-the-container).
+
+> [!TIP] If you run `molecule converge` on XNAT or OMERO scenarios, you can
+> access the XNAT web UI at `http://localhost:8080` and the OMERO web UI at
+> `http://localhost:8080/webclient`.
```

Comment:
not sure why we need to go to `http://localhost:8080/webclient` rather than `http://localhost:8080` to access the OMERO web ui (even though with curl we get a valid response from `http://localhost:8080`), so thought it was worth documenting here